main.go

```diff
@@ -32,9 +33,22 @@ func Handler(ctx context.Context, request events.APIGatewayProxyRequest) (string
 	}
 
 	if strings.HasPrefix(request.Body, "update=") {
-		err = sendMessage(request.Body)
-		if err != nil {
-			return "Error sending SNS message", err
+		sparqlQuery := sparql.NewQuery()
+		err = sparqlQuery.Parse(strings.NewReader(request.Body))
+
+		for _, part := range sparqlQuery.Parts {
+			if strings.ToUpper(part.Verb) == "INSERT" || strings.ToUpper(part.Verb) == "INSERT DATA" {
+				err = sendMessage("touch", strings.Join(uniqueSubjects(part.Graph), "\", \""), request.Body)
+				if err != nil {
+					return "Error sending SNS message", err
+				}
+			}
+			if strings.ToUpper(part.Verb) == "DELETE" || strings.ToUpper(part.Verb) == "DELETE DATA" {
+				err = sendMessage("delete", strings.Join(uniqueSubjects(part.Graph), "\", \""), request.Body)
```

Comment:
Related: I wonder how you would feel about adding test coverage to either/both the parse package below or the main package here?

Reply:
@mjgiarlo Doing that now, should be done before Standup.

Reply:
Thanks!